mdata/store_cassandra.go

```diff
@@ -73,13 +76,68 @@ It's safe for concurrent use by multiple goroutines and a typical usage scenario
 object to interact with the whole Cassandra cluster.
 */
 
+type ttlTables map[uint32]ttlTable
+type ttlTable struct {
+	table  string
+	window uint32
```

Comment:
in many places this is referred to as window_size. i suggest we call this windowSize to avoid confusion. plus it would make more sense. since the window would be "<window_size> hours", the window is not just a number of hours, whereas the window_size is